jazzmin/static/jazzmin/js/main.js

```diff
@@ -56,7 +56,14 @@
         const $changeListTable = $('#changelist .results table');
         if ($changeListTable.length && !$changeListTable.hasClass('table table-striped')) {
             $changeListTable.addClass('table table-striped');
-        }
+        };
+
+        var a_active = $('a.nav-link.active');
+        var main_li_parent = a_active.closest('li.nav-item.has-treeview');
+        var ul_child = main_li_parent.children('ul');
+
```

Comment:
please change `var`'s to `const` and use `$` prefixes for constants that contain jquery objects

Reply:
Maybe also move this code up into the `setActiveLinks` function

Reply:
done